### PR TITLE
since we do not depend on doctrine dbal 2.5+ yet, we also need to ignore DBALException

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php [hunk 1]
```diff
@@ -8,9 +8,10 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception\TableNotFoundException;
```
Comment (WouterJ) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: this one is no longer used?

File: src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php [hunk 2]
```diff
@@ -90,6 +91,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     if (false === $input->getOption('force')) {
                         throw $e;
                     }
+                // remove this once we require Doctrine DBAL 2.5+
```
Comment (WouterJ) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: "remove this" -> "change this to TableNotFoundException" ?

Comment (lsmith77) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: I left the ``TableNotFoundException``  . case in the code

Comment (WouterJ) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: ah, :+1: then

**Pull request merged**